mk/stack.mk

```diff
@@ -2,6 +2,4 @@ STACK=stack
 
 ifneq ($(wildcard $(TOP)/stack.yaml),)
   HAS_STACK := 1
-else
-  undefine HAS_STACK
```

Comment:
According to https://www.gnu.org/software/make/manual/html_node/Undefine-Directive.html the alternative to `undefine HAS_STACK` would be `HAS_STACK=` (set it to empty).  (Without looking more deeply into the make-structure:) Wouldn't this be a more conservative change?

Reply:
Yeah maybe. Fixed